turborepo-tests/example-with-svelte/package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "turborepo-examples-test-with-svelte",
+  "scripts": {
+    "test": "bash ../helpers/test_example.sh ../../examples/with-svelte pnpm"
```

Since the helpers package is installed, you can actually invoke this locally:

```suggestion
    "test": "bash ./node_modules/turborepo-tests-helpers/test_example.sh ../../examples/with-svelte pnpm"
```

(or use `npx`, `pnpx` or whatever if that lets you run a bash script)